src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
```

Comment:
This one adds n_bengs, b_per_fpga and beng_out_bits_per_sample. Should we add those 3 to the fxcorrelator.py and scrap this function?